src/App.tsx

```diff
@@ -1,26 +1,41 @@
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { TodoList } from './components/TodoList';
+import { TodoFilter } from './components/TodoFilter';
+import { TodoModal } from './components/TodoModal';
+import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { setTodos } from './features/todos';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App: React.FC = () => {
+  const [isLoading, setIsLoading] = useState(true);
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+  const selectedTodo = useAppSelector(state => state.todo);
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    getTodos()
+      .then(data => dispatch(setTodos(data)))
```

Comment:
better use async actions for fetching data
read more here https://redux-toolkit.js.org/api/createAsyncThunk